_health-care/_js/components/financial-assessment/ChildInformationSection.jsx

```diff
@@ -66,9 +71,10 @@ class ChildInformationSection extends React.Component {
         {notRequiredMessage}
 
         <div className="input-section">
-          <ErrorableCheckbox
+          <ErrorableRadioButtonGroup
```

Comment:
Now that this is a radio button, should this field be required? Can we assume if a user does not select either "yes" or "no" that they do not have children to report?